cmake/projects/OpenSSL/ep-stages/configure_1_1_plus.cmake.in

```diff
@@ -12,6 +12,20 @@ if("@openssl_dir@" STREQUAL "")
   message(FATAL_ERROR "openssl_dir is empty")
 endif()
 
+if("@MSVC@")
+  # Unset as we are already in a msvc environment 
+  # and the openssl perl script will figure things out.
+  # The created variable content by CMake is unqouted and will cause issues
+  unset(ENV{CC})
+  unset(ENV{CXX})
+  unset(ENV{RC})
+  unset(ENV{AR})
+  unset(ENV{RUNLIB})
+else()
+  set(log_opts "")
+endif()
+
+
 execute_process(COMMAND
   perl Configure @arch@ @opt@ @shared@ "--prefix=@openssl_install_dir@" "--openssldir=@openssl_dir@" RESULT_VARIABLE result)
```

Comment:
I've found that the perl command will fail depending on whether it is the mingw perl (failure) or the Strawberry/Windows perl (success). Can we fix that too?

Reply:
@rbsheth 
Sure
I did not look to close into it.
But it looks like the command expects a `linux bash`  but in fact it is a `windows cmd` 

Not yet sure how to solve that, but I'm also not so familiar with mingw and I need to set it up first.